consts/Cargo.toml

```diff
@@ -1,13 +1,10 @@
 [package]
 name = "edhoc-consts"
-version = "0.1.0"
+version.workspace = true
```

Comment:
Ahh! I was wondering if there was an easy way to keep all version numbers in sync. Great!